ArduCopter/tuning.cpp

```diff
@@ -9,8 +9,6 @@
 //  should be called at 3.3hz
 void Copter::tuning()
 {
-    const RC_Channel *rc_tuning = rc().find_channel_for_option(RC_Channel::AUX_FUNC::TRANSMITTER_TUNING);
```

Comment:
I guess a simpler solution might have been to add a call here to set_default_dead_zone() but that would be inconsistent with how we do it for other channels

Reply:
> I guess a simpler solution might have been to add a call here to set_default_dead_zone() but that would be inconsistent with how we do it for other channels

Yep, didn't want to answer the question, "why didn't you put it into `set_default_deadzones, that seems the right place?"...

As mentioned elsewhere, having the user need to reboot isn't a bad thing IMO.  If the user were to accidentally set the channel option to their roll channel and then away again then we'd now have a zero-default-deadzone on their roll channel.... and they could arm/takeoff in that state.